.evergreen/run-tests.sh

```diff
@@ -0,0 +1,97 @@
+    export GOPATH=$(cygpath -m $GOPATH)
+    export GOCACHE=$(cygpath -m $GOCACHE)
+
+    mkdir -p c:/libmongocrypt/include
+    mkdir -p c:/libmongocrypt/bin
+    curl https://s3.amazonaws.com/mciuploads/libmongocrypt/windows/latest_release/libmongocrypt.tar.gz --output libmongocrypt.tar.gz
+    tar -xvzf libmongocrypt.tar.gz
+    cp ./bin/mongocrypt.dll c:/libmongocrypt/bin
+    cp ./include/mongocrypt/*.h c:/libmongocrypt/include
+    export PATH=$PATH:/cygdrive/c/libmongocrypt/bin
+else
+    git clone https://github.com/mongodb/libmongocrypt
+    ./libmongocrypt/.evergreen/compile.sh
+fi
+
+export GOROOT="${GOROOT}"
+export PATH="${GOROOT}/bin:${GCC_PATH}:$GOPATH/bin:$PATH"
+export PKG_CONFIG_PATH=$(pwd)/install/libmongocrypt/lib/pkgconfig:$(pwd)/install/mongo-c-driver/lib/pkgconfig
+export LD_LIBRARY_PATH=$(pwd)/install/libmongocrypt/lib
+export GOFLAGS=-mod=vendor
+
+SSL=${SSL:-nossl}
+if [ "$SSL" != "nossl" ]; then
+    export MONGO_GO_DRIVER_CA_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/ca.pem"
+    export MONGO_GO_DRIVER_KEY_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client.pem"
+    export MONGO_GO_DRIVER_PKCS8_ENCRYPTED_KEY_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client-pkcs8-encrypted.pem"
+    export MONGO_GO_DRIVER_PKCS8_UNENCRYPTED_KEY_FILE="${DRIVERS_TOOLS}/.evergreen/x509gen/client-pkcs8-unencrypted.pem"
+
+    if [ "Windows_NT" = "$OS" ]; then
+        export MONGO_GO_DRIVER_CA_FILE=$(cygpath -m $MONGO_GO_DRIVER_CA_FILE)
+        export MONGO_GO_DRIVER_KEY_FILE=$(cygpath -m $MONGO_GO_DRIVER_KEY_FILE)
+        export MONGO_GO_DRIVER_PKCS8_ENCRYPTED_KEY_FILE=$(cygpath -m $MONGO_GO_DRIVER_PKCS8_ENCRYPTED_KEY_FILE)
+        export MONGO_GO_DRIVER_PKCS8_UNENCRYPTED_KEY_FILE=$(cygpath -m $MONGO_GO_DRIVER_PKCS8_UNENCRYPTED_KEY_FILE)
+    fi
+fi
+
+# Set temp credentials for AWS if python3 is available.
+#
+# Using python3-venv in Ubuntu 14.04 (an OS required for legacy server version
+# tasks) requires the use of apt-get, which we wish to avoid. So, we do not set
+# a python3 binary on Ubuntu 14.04. Setting AWS temp credentials for legacy
+# server version tasks is unneccesary, as temp credentials are only needed on 4.2+.
+if [ ! -z ${PYTHON3_BINARY} ]; then
+  export AWS_ACCESS_KEY_ID="${cse_aws_access_key_id}"
+  export AWS_SECRET_ACCESS_KEY="${cse_aws_secret_access_key}"
+  export AWS_DEFAULT_REGION="us-east-1"
+  ${PYTHON3_BINARY} -m venv ./venv
+
+  # Set the PYTHON environment variable to point to the active python3 binary. This is used by the
+  # set-temp-creds.sh script.
+  if [ "Windows_NT" = "$OS" ]; then
+    export PYTHON="$(pwd)/venv/Scripts/python"
+  else
+    export PYTHON="$(pwd)/venv/bin/python"
+  fi
+
+  ./venv/${VENV_BIN_DIR|bin}/pip3 install boto3
+  . ${DRIVERS_TOOLS}/.evergreen/csfle/set-temp-creds.sh
+fi
+
+# If GO_BUILD_TAGS is not set, set the default Go build tags to "cse" to enable
+# client-side encryption, which requires linking the libmongocrypt C library.
+if [ -z ${GO_BUILD_TAGS+x} ]; then
```

Comment:
I think this may have been a typo in https://github.com/mongodb/mongo-go-driver/pull/663/files#diff-2bc841e86ce96b7b422ae203fd8315d0b2a461956cecbe0e096420656fc3fb12R320
```suggestion
if [ -z ${GO_BUILD_TAGS} ]; then
```

Reply:
I actually looked this up! It lets us differentiate between unset and set to null, and we don't want to do this if its set to null on purpose